### ISSUE TITLE
Remove the use of the legacy Patternfly table

### ISSUE THREAD

**Issue by edewit** [keycloak/keycloak-admin-ui#1236]
The legacy patternfly table is harder to test as we don't have any control over how the rows are rendered.

**Comment by ssilvert** [comment]
@edewit Can this be closed?  If so, please close it.  If not, please leave a comment with a little more detail on how and when we can address the problem.

**Comment by edewit** [comment]
We can't close it as we didn't do it yet, basically instead of using this [Patternfly table component]( https://www.patternfly.org/v4/components/table/react-legacy) we should use the [new one](https://www.patternfly.org/v4/components/table) instead it's a refactoring issue.

**Comment by ssilvert** [comment]
OK.  I added the `missing/tests` tag just because it sounds like this is something we need to do as we build out the test suite.

**Comment by jonkoops** [comment]
I don't think this has anything to do with tests, it's just to remove legacy API.

**Comment by ssilvert** [comment]
Defer.
